### DISCUSSION TITLE
いいね数が多く、テーマに即してもいる記事を検索する導線の乏しさについて

### DISCUSSION THREAD

**Discussion by honey32** [increments/qiita-discussions#503] [要望・アイデア・不具合]
単刀直入に言うと、 Zenn 様の「あるタグの Alltime タブ」に表示されるような記事へのアクセスのしづらさについてです。

このような記事、つまり一時的に人目を惹くだけでなく、その技術が生きているかぎり参照されつづけられるような高品質な記事（仮に「ストック性のある記事」と呼びます）が Qiita に多く存在すると思いますが、

Google 検索からならそれらを見つけられるものの、Qiita のサイト内を回遊してそのような記事を見つけるのが困難です。

ストック性に関わらず「トレンド」タブには「一時的に多くの人に評価された(フロー的な)」記事が表示されることに比べると、ストック性のある古い記事はサイトのUIの中では埋もれてしまっている印象を受けます

## 現状の詳細

「React」のような広範的なテーマについて、一定の質がありそうな記事を見つける方法は、以下のリストのように複数あると認識しています。しかし、どれも詳細なクエリの機能を知らなければありませんし、実際に私はこの discussion を立てるに当たって、久しく使っていなかったその機能を久々に使いました。

これでは初学者の方にとって、古くて質の高い記事を見つけるのが困難であり、「初心者向けに良い記事が書かれても目に入りにくいので、記事の良し悪しの判断が困難な彼らの評価が、質で劣る記事に流れてしまい」、良い記事を書いた人が正当に評価されず、初心者が良い記事を見つけられず損をする場になっているのではないか、と危惧しています。

また、第一印象として質が高く古い記事が目につかず、多くの人に「質の悪い記事が多い」印象を与えているかもしれません。

### タグ詳細ページ

例: https://qiita.com/tags/react

- 「いいね数によるソート」ができない
- 「トレンド」欄もありまるが、件数が 5 件と少なすぎる
  - あと、フロー・ストックが同じ箇所に押し込まれている感じもするのでなおさらです。

### 検索ページ

- 「tag:」クエリを書いて「いいね数」でソートすればストック性の記事を見つけられる
  - https://qiita.com/search?q=tag%3Areact%20&sort=like
- 「tag:」クエリを知らない人が、キーワード検索で<br/> **「関連度順」** でソートした場合
  - 中身のあまり濃くないページ（初学者の方の「概要を調べてみた」記事）が上位ヒットする
    - もしかして、アルゴリズムがたまたまフィットしてしまっている？
    - ストック数でフィルターすればいいんでしょうけど
      - https://qiita.com/search?sort=rel&q=react++stocks%3A%3E50
- 「tag:」クエリを知らない人が、キーワード検索で<br/> **「いいね数」** でソートした場合
  - 関連度が低い記事が出てくるので、知りたい情報が手に入らない。

## あとがき

Qiita 様あってこそ今の自分がありますし、恩義はありますが、正直なところ高品質な記事を作るモチベーションについては Zenn に傾ききっており、複雑な心境です。

このようなことは既にご承知であれば釈迦に説法かもしれませんが、Qiita 様も「気軽に投稿できる」だけでなく「初学者が確実に良い記事を見つけることができる」「良い記事を書いた人が正当に評価されたと思える」コミュニティであってほしいと思っております。

**Comment by tanaka-asami415** [comment]
@honey32 
詳細にご記載いただきありがとうございます！
気づけていなかった内容はもちろんですが、課題の解像度が上がりました🙇

どのような形にできるか、具体的なイメージはできていないのですが、
「初学者の方でも良い記事を見つけやすい」「良い記事が埋もれない」ような機能を検討したいと思います。

サービス改善に努めてまいりますので、今後ともQiitaをご愛顧いただけますと幸いです。

**Reply by honey32** [reply]
ありがとうございます。
